### PR TITLE
Add Movie endpoint

### PR DESCRIPTION
**Pull request by Perpluu**
- Added movie endpoint
- Added movie tests
- Added home-page tests

### CODE REVIEW

File: views.py [hunk 1]
```diff
@@ -13,14 +13,13 @@ async def read_number_of_planets():
         return {"Number of Planets": planets["count"]}
 
 
-@app.post("/movie")
+@app.post("/movie/{movie_id}")
 async def get_movie_data(movie_id: int):
     response = requests.get(f"https://swapi.py4e.com/api/films/{movie_id}/")
```
Comment (mociepka) [inline views.py]: Please use only one lib for requests. Requests lib is not asynchronous. You already used httpx lib above. Please be consistent.

File: views.py [hunk 2]
```diff
@@ -13,14 +13,13 @@ async def read_number_of_planets():
...
-    if response.status_code != 200:
-        return {"error": f"Movie with ID {movie_id} not found."}
     movie_data = response.json()
     people = [
```
Comment (mociepka) [inline views.py]: This line will wait synchronously for all requests to finish. Please use asynchronous lib httpx with chain https://realpython.com/async-io-python/#chaining-coroutines

File: conftest.py [hunk 3]
```diff
@@ -1,5 +0,0 @@
-from pytest_socket import disable_socket
-
-
-# def pytest_runtest_setup():
```
Comment (mociepka) [inline conftest.py:4]: Remove commented code.

**Review by mociepka — COMMENTED** [review]
Please ignore all hidden files line .coverage, amd __pycache__

File: views.py (hunk 1 above)
Comment (Perpluu) [reply views.py]: Fixed

File: views.py (hunk 2 above)
Comment (Perpluu) [reply views.py]: Fixed

File: conftest.py (hunk 3 above)
Comment (Perpluu) [reply conftest.py:4]: Fixed

**Pull request merged**